backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -237,58 +250,153 @@ void getPatients_noFacility_allFetchedAndSorted() {
     assertPatientList(patients, CHARLES, BRAD, ELIZABETH, AMOS);
-    patients = _service.getPatients(_site2.getInternalId(), PATIENT_PAGEOFFSET, PATIENT_PAGESIZE);
+    patients =
+        _service.getPatients(
+            _site2.getInternalId(), PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, false, null);
     assertPatientList(patients, FRANK, BRAD, DEXTER, AMOS);
   }
 
   @Test
   @WithSimpleReportOrgAdminUser
   void getPatients_pagination() {
     makedata(true);
-    List<Person> patients_org_page0 = _service.getAllPatients(0, 5);
-    List<Person> patients_org_page1 = _service.getAllPatients(1, 5);
-    List<Person> patients_org_page2 = _service.getAllPatients(2, 5);
-    List<Person> patients_org_page3 = _service.getAllPatients(3, 5);
+    List<Person> patients_org_page0 = _service.getPatients(null, 0, 5, false, null);
+    List<Person> patients_org_page1 = _service.getPatients(null, 1, 5, false, null);
+    List<Person> patients_org_page2 = _service.getPatients(null, 2, 5, false, null);
+    List<Person> patients_org_page3 = _service.getPatients(null, 3, 5, false, null);
 
     assertPatientList(patients_org_page0, CHARLES, FRANK, JANNELLE, BRAD, DEXTER);
     assertPatientList(patients_org_page1, KACEY, ELIZABETH, LEELOO, AMOS, IAN);
     assertPatientList(patients_org_page2, HEINRICK, GALE);
     assertEquals(0, patients_org_page3.size());
 
-    List<Person> patients_site2_page0 = _service.getPatients(_site2.getInternalId(), 0, 4);
-    List<Person> patients_site2_page1 = _service.getPatients(_site2.getInternalId(), 1, 4);
-    List<Person> patients_site2_page2 = _service.getPatients(_site2.getInternalId(), 2, 4);
+    List<Person> patients_site2_page0 =
+        _service.getPatients(_site2.getInternalId(), 0, 4, false, null);
+    List<Person> patients_site2_page1 =
+        _service.getPatients(_site2.getInternalId(), 1, 4, false, null);
+    List<Person> patients_site2_page2 =
+        _service.getPatients(_site2.getInternalId(), 2, 4, false, null);
 
     assertPatientList(patients_site2_page0, FRANK, JANNELLE, BRAD, DEXTER);
     assertPatientList(patients_site2_page1, KACEY, LEELOO, AMOS);
     assertEquals(0, patients_site2_page2.size());
   }
 
+  @Test
+  @WithSimpleReportOrgAdminUser
+  void getPatients_search_OrgAdminUser() {
+    makedata(true);
+
+    // delete some data to verify achived works as expepected
+    // delete Charles (_site1)
+    Person charles = _service.getPatients(null, 0, 5, false, null).get(0);
+    _service.setIsDeleted(charles.getInternalId(), true);
+    // Delete Frank (_site2)
+    Person frank = _service.getPatients(_site2.getInternalId(), 0, 5, false, null).get(0);
+    _service.setIsDeleted(frank.getInternalId(), true);
+
+    // all facilities, not deleted, "ma"
+    List<Person> patients = _service.getPatients(null, 0, 100, false, "ma");
+    assertPatientList(patients, JANNELLE, KACEY, ELIZABETH, HEINRICK, GALE);
```

Comment:
is there some way to have some mapping of person to site or vice versa, without having to look at makedata()? this block is a little hard to understand.

Reply:
I agree. These unit tests has gotten more complicated as I've added pagination and now searching to `PersonService`, but I hate refactoring all the unit tests in this file and NOT making any mistakes. (The existing people constants are used by multiple tests here).

I figured it was safer to leave existing unit tests in place while I make this change. The risk is that I'll break some existing functionality, but also refactor the existing tests "fixing" them to work without realizing I actually changed their behavior.